src/function.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable*/
+let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+
+  function clearAllSelected() {
+    let newTasks = tasks.filter(function(task) {
+      return task.completed != true;
+    })
+
+    for (let i = 0; i < newTasks.length; i += 1) {
+     newTasks[i].index = i + 1;
+   }
+
+     localStorage.setItem('tasks', JSON.stringify(newTasks));
+     window.location.reload();
+  }
+
+  function returnNewArrar (event) {
+    console.log(event);
+      const target1 = event.target.id;
+      console.log(target1);
+      const targetIdNumber =parseInt(target1.match(/\d+/)[0], 10);
+      console.log(JSON.stringify(targetIdNumber));
+      const taskIndex = tasks.findIndex(task => task.index === targetIdNumber + 1);
+      console.log(taskIndex);
+      tasks[taskIndex].completed = true;
+      const nextElemSinling1 = event.target.nextElementSibling;
+      console.log(nextElemSinling1);
+      nextElemSinling1.style.textDecoration = 'line-through' ;
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
```

Comment:
- [ ] Great job working on the change the task to true when the user press on the check box but it would be great if you implement the opposite as well so it toggles the true to be false if the user press again I think in your case when I press it turns to true that's right but if I press again it just keep true not turn to false but it is very important to toggle it as mentioned in the requirements 💪🏻🤩🔥💯


![image](https://github.com/Nahid1911/WebPack_To_Do_List/assets/58553711/bef4be4c-c5a5-41a3-af5c-8fc6cd90fb55)

Reply:
@Reem-lab 
functionality added